posters-overrides.json

```diff
@@ -3338,7 +3338,14 @@
    "number": 1184
   },
   {
-   "number": 1186
+   "number": 1186,
+   "title": "Permutation Inference for Canonical Correlation Analysis (CCA)",
+   "institution": ["National Institutes of Health (NIH)", "University of Geneva", "University of Oxford"],
```

Comment:
I'm not sure if the code is set up to handle string fields being overridden with lists, so I will leave further review to those more involved in the recent changes.

Reply:
yes, please limit only to PDF URL -- affiliation of the speaker already picked up, all the authors as well.

Reply:
I just sent https://github.com/datalad-datasets/ohbm2020-posters/pull/129 that can handle institution field to be either string or array of strings.